src/Agent.Listener/JobDispatcher.cs

```diff
@@ -389,6 +389,10 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, WorkerDisp
                                 requireExitCodeZero: false,
                                 outputEncoding: null,
                                 killProcessOnCancel: true,
+                                redirectStandardIn: null,
```

Comment:
I just copied the defaults here. It would also work to add a new ExecuteAsync overload with the exact parameters + the new `highPriorityProcess`